ruff.toml

```diff
@@ -0,0 +1,22 @@
+[lint]
+extend-ignore = [
+	# https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+	"W191",
+	"E111",
+	"E114",
+	"E117",
+	"D206",
+	"D300",
+	"Q000",
+	"Q001",
+	"Q002",
+	"Q003",
+	"COM812",
+	"COM819",
+	"ISC001",
+	"ISC002",
+]
+
+[format]
+# https://docs.astral.sh/ruff/settings/#format-quote-style
+quote-style = "preserve"
```

Comment:
This requires the `preview` flag (which you can turn on in configs, otherwise it just forces typing an extra flag to run ruff in CLI, and extra editor configs to run in IDE). https://docs.astral.sh/ruff/preview/
![image](https://github.com/jaraco/skeleton/assets/1350584/2de777eb-5d36-4ef9-a4b5-7bb38b6b88a0)
![image](https://github.com/jaraco/skeleton/assets/1350584/e598c054-a9ee-4ec6-a123-fccd6e1d816f)

Reply:
Indeed, got bitten by that in other projects meanwhile. I see no alternative to `preview` apart from choosing a different `quote-style` setting. So i suggest #108.